pylot/operator_creator.py

```diff
@@ -443,6 +443,7 @@ def add_mpc_agent(can_bus_stream, waypoints_stream):
 
 def add_pid_agent(can_bus_stream, waypoints_stream):
     op_config = erdos.OperatorConfig(name='pid_agent_operator',
+                                     flow_watermarks=False,
```

Comment:
Can you please also add flow_watermarks=False to the SynchronizerOperator and the MPCAgentOperator?

Reply:
Yeah sure

Reply:
It's already on the synchronizer operator for me, I just added it to the MPC agent op.